test/libsolidity/ASTJSON/userDefinedValueType.json

```diff
@@ -291,15 +291,15 @@
               "typeDescriptions":
               {
                 "typeIdentifier": "t_userDefinedValueType$_MyAddress_$18",
-                "typeString": "user defined type MyAddress"
+                "typeString": "user defined type C.MyAddress"
```

Comment:
Why is it only updated here and not in cases above where it still doesn't use the FQN?

Reply:
The others are in global scope. But not sure what you mean.

Reply:
FQN = Fully Qualified Name

Reply:
The changes to ABIJson file was missing. Added it now.